clients/python/moondream/__init__.py

```diff
@@ -1,5 +1,18 @@
 from .types import VLM
 from .onnx_vl import OnnxVL
+from .cloud_vl import CloudVL
+
+
+def VL(
+    model_path: str = None,
+    api_key: str = None,
+    api_url: str = None,
+    version: str = "v1",
```

Comment:
Remove these. We can always add options in the future when they're needed, but removing an existing argument breaks backward compatibility.